hugo/content/grafana/_index.md

```diff
@@ -106,3 +107,44 @@ The extras package takes care of putting all these files in place. If you did no
 [paths]
 provisioning = /etc/grafana/provisioning
 ```
+
+### Included Dashboards {#dashboards}
+
+pgMonitor comes with several dashboards ready to be used with automatic provisioning. They provide examples of using the metrics from the postgres_exporter and node_exporter. Since provisioned dashboards cannot be edited directly in the web interface, if any custom changes are desired, it is recommmended to make a copy of them and make your changes there.
+
+#### PostgreSQL Dashboards
+
+ * *Bloat Details* - Provides details on database bloat (wasted space). Provides overview and top-n statistics.
```

Comment:
also show dashboard filename ?
A table might look better.

Reply:
I'd had the filename before, but it looked weird in this format. Will try with a table.

Reply:
Turned into table and added filenames. The filenames are almost exactly the same as the dashboard titles, so does look a little redundant. Can leave it in of consensus likes it tho.

Reply:
Looks nice with table, my 2 cents filename are good to have.
Other ideas
- move this table to the top just after intro section
- Link each dashboard to a scaren shot

Reply:
Moved the table to the top. 

Can see about adding more pictures in the future if still desired. Does mean having to keep up with design changes and appearance does change slightly with new major grafana versions.